tests/testthat/test_multisynth.R

```diff
@@ -77,7 +77,9 @@ test_that("Separate synth is the same as fitting separate synths", {
                        scm=T, eps_rel=1e-5, eps_abs=1e-5)
     
     # weights are the same-ish
-    expect_equal(c(scm17$weights), c(msyn$weights[-c(15, 16), 2]), tolerance=3e-2)
+    sscm_weights <- unname(c(scm17$weights))
+    mscm_weights <- unname(c(msyn$weights[-c(15, 16), 2]))
+    expect_equal(sscm_weights, mscm_weights, tolerance=3e-2)
```

Comment:
Could you add a check that the names match up (modulo the treated unit)? That'd make sure nothing funky is happening.

Reply:
Added.